lib/modules/manager/helmfile/artifacts.ts

```diff
@@ -59,9 +66,32 @@ export async function updateArtifacts({
         constraint: config.constraints?.kustomize,
       });
     }
-    await exec(`helmfile deps -f ${quote(packageFileName)}`, {
+
+    const cmd: string[] = [];
+    const doc = parseDoc(newPackageFileContent);
+
+    const regexOfURLPath = /\/.*/;
+    const repositories = getRepositories(doc);
+    if (repositories.length < 1) {
+      return null;
+    }
```

Comment:
this seems wrong to me. why should we not run lockfile update without repositories?

Reply:
I didn't understand helmfile diff very well. helmfile.lock can be updated if repositories are defined in helmfileDefault., so it should return updated helmfile.lock, not null. The result should return an updated helmfile.lock instead of null.